README.md

```diff
@@ -1,84 +1,90 @@
-# `jest-runner-tsd`
+# jest-runner-tsd
 
-A Jest runner that tests typescript typings using [tsd](https://github.com/SamVerschueren/tsd) under the hood.
+> Run your TypeScript type tests using Jest.
 
-## Install
+[![version](https://img.shields.io/npm/v/jest-runner-tsd.svg)](https://npmjs.com/package/jest-runner-tsd)
+[![license](https://img.shields.io/github/license/jest-community/jest-runner-tsd.svg)](https://github.com/jest-community/jest-runner-tsd/blob/main/LICENSE.md)
 
-Install `jest-runner-tsd`
+**Note:** the `jest-runner-tsd` is using [`tsd-lite`](https://github.com/mrazauskas/tsd-lite) instead of [`tsd`](https://github.com/SamVerschueren/tsd). Both of them have the same type testing logic, but `tsd-light` makes it easier to test projects written in TypeScript (or types generated by your library).
 
-```bash
-yarn add --dev jest-runner-tsd
+Most important differences (for the full list see [`tsd-lite` repo](https://github.com/mrazauskas/tsd-lite)):
+
+- `tsd-lite` has no additional [rules](https://github.com/SamVerschueren/tsd/issues/32) or checks;
+- `jest.config` is used to discover test files;
+- and `tsconfig.json` provides configuration for TS compiler. For details see [Configuration](#configuration) section.
 
-# or with NPM
+## Install
 
-npm install --save-dev jest-runner-tsd
+```bash
+yarn add -D jest-runner-tsd @tsd/typescript
 ```
 
-### Adding to Jest Config
+Remember to install `@tsd/typescript` package. It is a required peer dependency.
+
+## Configuration
 
-Create a `jest.config.types.js` file and have the runner property set to `jest-runner-tsd` as shown below:
+### Jest
+
+First of all, you should [configure Jest](https://jestjs.io/docs/configuration) to discover your test files. For example, if the files are suffix with `.test.ts` and live inside `__typetests__` directory, set up `jest.config.tsd.js` like this:
 
 ```js
 module.exports = {
+  displayName: {
+    color: 'blue',
+    name: 'types',
+  },
   runner: 'jest-runner-tsd',
+  testMatch: ['**/__typetests__/*.test.ts'],
 };
 ```
 
-In the project `package.json` file, modify the scripts block to use the configuration file as show below:
+### TS Compiler
 
-```json
-...
-"scripts": {
-  ...
-  "type-tests": "yarn jest --config jest.config.types.js"
-}
-...
-```
+Your test files will be compiled using the [TypeScript compiler](https://github.com/microsoft/TypeScript/wiki/Using-the-Compiler-API) (similarly to `tsc`), but, instead of emitting JS code, `tsd-lite` will analyze types and diagnostics returned by the compiler.
```

Comment:
```suggestion
Your test files will be compiled using the [TypeScript compiler](https://github.com/microsoft/TypeScript/wiki/Using-the-Compiler-API) (similar to `tsc`), but, instead of emitting JS code, `tsd-lite` will analyze types and diagnostics returned by the compiler.
```